src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/ElasticSearchTrialDAOSpec.scala

```diff
@@ -120,19 +120,19 @@ class ElasticSearchTrialDAOSpec extends FreeSpec with Matchers with BeforeAndAft
       }
     }
     "projectReport" - {
-      "should return an accurate report with only verified/claimed projects" in {
+      "should return an accurate report with all verified/ready projects" in {
         // unverified/unclaimed projects listed below but commented out for developer clarity
```

Comment:
I couldn't understand why the lines that are commented out were left over.

Reply:
I'll fix that ... it's carry over from before I changed what `projectReport` does. That change was one of David's suggestions.

Reply:
... re-read your comment. They were left there intentionally by David so other developers would be able to see all of the potential trial projects that could be returned. I will update the comment to be more clear.